<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreDemande</class>
 <widget class="QWidget" name="FenetreDemande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Autorisation chef service :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="6">
      <widget class="QTextEdit" name="txt_description_demande">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Description :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QLineEdit" name="txt_id_personnel">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7" colspan="2">
      <widget class="QPushButton" name="btn_demande_pdf">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(36, 198, 0);</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Objet du congé :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7" colspan="2">
      <widget class="QPushButton" name="btn_parametres">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Paramètres</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7" colspan="2">
      <widget class="QPushButton" name="btn_supprimer">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 152, 152);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de fin :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7" colspan="2">
      <widget class="QPushButton" name="btn_ajouter">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(36, 198, 0);</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7" colspan="2">
      <widget class="QPushButton" name="btn_modifier">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(188, 195, 255);</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="4">
      <widget class="QLineEdit" name="txt_chercher">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(132, 8, 255);</string>
       </property>
       <property name="text">
        <string>Demandes</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox_autorisation_chefservice">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Oui</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Non</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="5" colspan="2">
      <widget class="QPushButton" name="btn_chercher">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Chercher demande</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="btn_afficher_tout">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Afficher tout</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="9">
      <widget class="QTableWidget" name="tableWidget_affichage">
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID Personnel</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Objet</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date de début</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date de fin</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Autorisation chef service</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Demande validé</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date de création</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date de modification</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="txt_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QDateEdit" name="date_debut">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QDateEdit" name="date_fin">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="6">
      <widget class="QComboBox" name="comboBox_objet_demande">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID Personnel :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5" colspan="2">
      <widget class="QPushButton" name="btn_chercher_employe">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Chercher employé</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de début :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7" colspan="2">
      <widget class="QPushButton" name="btn_nouveau">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 240, 195);</string>
       </property>
       <property name="text">
        <string>Nouveau</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_id</tabstop>
  <tabstop>txt_id_personnel</tabstop>
  <tabstop>btn_chercher_employe</tabstop>
  <tabstop>date_debut</tabstop>
  <tabstop>date_fin</tabstop>
  <tabstop>comboBox_objet_demande</tabstop>
  <tabstop>btn_nouveau</tabstop>
  <tabstop>btn_ajouter</tabstop>
  <tabstop>btn_modifier</tabstop>
  <tabstop>btn_parametres</tabstop>
  <tabstop>btn_supprimer</tabstop>
  <tabstop>btn_afficher_tout</tabstop>
  <tabstop>txt_chercher</tabstop>
  <tabstop>tableWidget_affichage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
